docs/reference/queries.txt

```diff
@@ -1886,9 +1886,7 @@ in the default scope, the value in the default scope takes precedence:
 
   Band.new # => #<Band _id: 5c3f74ddce4ef3791abbb088, name: nil, active: false>
 
-Because a default scope initializes fields in new models as just described,
-defining a default scope with a dotted key and a simple literal value is
-not possible:
+As of Mongoid 8, a default scope can be defined with a dotted key:
```

Comment:
When I first read this, I thought it was referring to dot/dollar key names. I'd recommend to be more explicit like the following:

```
As of Mongoid 8, a default scope can be defined using dot-notation for path traversal:
```